ui/src/routes/notes/[teamNumber]/+page.svelte

```diff
@@ -1,7 +1,50 @@
-<script>
+<script lang="ts">
 	import { page } from '$app/stores';
+	import TeamMatchNote from '$lib/components/notes/TeamMatchNote.svelte';
+	import type { components } from '../../../fms/fms-api';
+
+	type MatchNoteModel = components["schemas"]["MatchNoteModel"];
 
 	const params = $page.params;
+
+	const notes: MatchNoteModel[] = [
+	{
+				note: "Hungry for carpet",
+				noteId: "1",
+				tournamentLevel: "Practice",
+				matchNumber: 1,
+				playNumber: 1,
+				teamNumber: parseInt(params.teamNumber!),
+				timeAdded: "Wed Aug 7 2024 00:00:00 GMT+0000 (Coordinated Universal Time)",
+				timeUpdated: "Wed Aug 7 2024 00:00:00 GMT+0000 (Coordinated Universal Time)",
+				isDeleted: false,
+				whoAdded: "", // TODO will this be a usable format?
```

Comment:
We can look at updating the model, but we'll need to coordinate with @kerrydh if we do make any changes, as he'll likely need to update the legacy app.

Reply:
Not necessarily looking to update the model, I just want to get see what this looks like in practice. I'm _assuming_ that it'll be the FTA's user ID, just want to confirm.